cmd/engine/main.go

```diff
@@ -888,25 +876,6 @@ func parseBoolOrAuto(s string) (*bool, error) {
 	return &b, err
 }
 
-// Run a separate gRPC serving _only_ the trace/log exporter services.
-func runOtelController(p string, pubsub *telemetry.PubSub) error {
```

Comment:
This isn't needed anymore :tada: - now we just send OTLP over HTTP and the requests get sent straight to the PubSub, instead of mounting in a unix socket and setting up a TCP proxy.